public/data/leabuende.json

```
@@ -0,0 +1,13 @@
+{	
+    "name": "Léa Buendé",	
```

```suggestion
    "name": "Lea Buende",	
```
Can you change the letter `e` to simple form. As you will be easy to search on searchbar.

---

Done, thank you @shyamtawli  !